packages/p2p-v2/src/components/Modals/PaymentMethods/AddEditPaymentMethodErrorModal/AddEditPaymentMethodErrorModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import ReactModal from 'react-modal';
+import { Button } from '@deriv-com/ui/dist/components/Button';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import { customStyles } from '../../helpers';
+import './AddEditPaymentMethodErrorModal.scss';
+
+type TAddEditPaymentMethodErrorModalProps = {
+    errorMessage: string;
+    isOpen: boolean;
+    onComfirm: () => void;
+};
+
+const AddEditPaymentMethodErrorModal = ({ errorMessage, isOpen, onComfirm }: TAddEditPaymentMethodErrorModalProps) => {
+    useEffect(() => {
+        ReactModal.setAppElement('#v2_modal_root');
+    }, []);
```

Comment:
```suggestion
    useEffect(() => {
        ReactModal.setAppElement('#v2_modal_root');
    }, []);
```

Maybe this should be added to the `src/App.tsx` or `src/routes/Home/index.tsx` files instead of adding them to every `Modal`, since it's not `instance` specific 🤔

Reply:
did this work?